{"name": "test_fail", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_fail():\n>       assert False\nE       assert False\n\ntests/test_simple/test_fail.py:6: AssertionError"}, "start": 1685201585277, "stop": 1685201585277, "uuid": "6f8153a3-e2df-4043-92cc-3805d0b910c0", "historyId": "92af0fa1762daaa99602acea1e86e189", "testCaseId": "92af0fa1762daaa99602acea1e86e189", "fullName": "tests.test_simple.test_fail#test_fail", "labels": [{"name": "parentSuite", "value": "tests.test_simple"}, {"name": "suite", "value": "test_fail"}, {"name": "host", "value": "MacBook-Air-Lasha.local"}, {"name": "thread", "value": "75677-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_simple.test_fail"}]}